### ISSUE TITLE
List update failed

### ISSUE THREAD

**Issue by technobecet** [MultiMC/MultiMC5#2605]
System Information
-----------------------------
MultiMC version: MultiMC 5 - Version 0.6.5-1217

Operating System:  KDE Neon 5.15 x64

Summary of the issue or suggestion:
----------------------------------------------
List update failed:
Job 'Download of meta file net.minecraft/index.json' failed to process:
https://v1.meta.multimc.org/net.minecraft/index.json

What should happen:
------------------------------
openssl1.0 and  libssl1.0-dev packages installed.
 ca-certificates
 ca-certificates-java
 ca-certificates-mono
ca-cacert
installed

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.download current version of multimc

2.install

3.open

Suspected cause:
---------------------------

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
![multimc](https://user-images.githubusercontent.com/2773780/56133434-0929a200-5f95-11e9-8ea9-3a6b415aca72.png)
[MultiMC-0.log](https://github.com/MultiMC/MultiMC5/files/3080150/MultiMC-0.log)

Additional Info:
---------------------------

**Comment by peterix** [comment]
Possibly related:
https://lutris.net/faq#ssl-issue

**Comment by TedsterTech** [comment]
Beep,
I managed to fix this on a friend's Pop!_OS machine (based on Ubuntu 19.04) by:
- Removing the Qt5 library files in the .share/multimc/bin or whatever directory
- 
- Installing the libraries required for those Qt5 files (as complained about not being there by MultiMC) as packages
- Symblinking the Qt5 files into the Bin dir. Copying them should've been about the same.

I also tried things like installing alternative SSL things, but I don't think they made the difference.
So, something like this should probably fix (on debian-based):

`cd .share/multimc
sudo apt install libqt5svg5 libqt5xml5
rm bin/libQt5Svg.so.5 bin/libQt5Xml.so.5
ln -s /usr/lib/x86_64-linux-gnu/libQt5Svg.so.5 bin/libQt5Svg.so.5
ln -s /usr/lib/x86_64-linux-gnu/libQt5Xml.so.5 bin/libQt5Xml.so.5
./MultiMC`

(no guarantees that these commands will actually work/that the filepaths are accurate 🐸)

Hope this helps
